install.sh

```diff
@@ -132,8 +132,9 @@ get_release_version() {
         version=${INSTALL_RKE2_VERSION}
     else
         info "finding release for channel ${INSTALL_RKE2_CHANNEL}"
-        # version_url="${INSTALL_RKE2_CHANNEL_URL}/${INSTALL_RKE2_CHANNEL}"
-        version_url="${INSTALL_RKE2_GITHUB_URL}/releases/latest"
+        INSTALL_RKE2_CHANNEL_URL=${INSTALL_RKE2_CHANNEL_URL:-'https://update.rke2.io/v1-release/channels'}
+        INSTALL_RKE2_CHANNEL=${INSTALL_RKE2_CHANNEL:-'stable'}
+        version_url="${INSTALL_RKE2_CHANNEL_URL}/${INSTALL_RKE2_CHANNEL}"
```

Comment:
This looks fine but does it leave `INSTALL_RKE2_GITHUB_URL` cruft? If so we should remove that.

Reply:
Nevermind, I see that `$INSTALL_RKE2_GITHUB_URL` is used for the downloads.

Reply:
it's used for actually downloading tarball and checksum after detecting the desired version from channelserver